s3fs/core.py

```diff
@@ -943,6 +944,12 @@ async def _put_file(self, lpath, rpath, chunksize=50 * 2 ** 20, **kwargs):
             else:
                 await self._mkdir(lpath)
         size = os.path.getsize(lpath)
+
+        content_type, _ = mimetypes.guess_type(lpath)
+
+        if content_type is not None:
+            kwargs.setdefault("ContentType", content_type[0])
```

Comment:
`content_type` is already unpacked, so AFAIK this should be:
```suggestion
            kwargs.setdefault("ContentType", content_type)
```